ang/civicase/CaseDetails--tabs--summary.html

```diff
@@ -44,6 +44,78 @@
     </div>
   </div>
   <!-- End - Subject and QuickLinks -->
+  <div class="civicase__summary-tab-row clearfix">
+    <div class="civicase__summary-tab-tile col-lg-6">
+      <!-- next milestone card -->
+      <h3> Next Milestone </h3>
+      <div class="civicase__activity-card civicase__activity-card--big">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <!-- ActivityICon directive -->
+            <span class="civicase__activity-icon-container civicase__activity-icon-container--ribbon">
+              <span>
+                <i class="text-danger civicase__activity-icon fa fa-bell"></i>
+              </span>
+              <!-- Activity Ribbon -->
+              <span class="text-danger civicase__activity-icon-ribbon"></span>
+            </span>
+            <!-- ActivityICon directive End -->
+            <!-- Activity Body -->
+            <div class="civicase__activity-card-inner">
```

Comment:
This class is exclusive to `big` card right? in that case class name should show that. Also as you have commented body, maybe its better to call it the same?
`civicase__activity-card--big__body`

Please also change any other classes which are exclusive to Big, to have the `--big`.

Reply:
As per @reneolivo suggestion - https://github.com/compucorp/uk.co.compucorp.civicase/pull/28#discussion_r218457351 . The generic styles and classes which are not specific to the big variant should be considered separately which is not incorrect too.

Going forward we need to discuss and come on a common consensus for the BEM naming conventions. Lets discuss this in a call.